TheoryTesting.ipynb

```diff
@@ -105,7 +106,7 @@
     "        n_measurements=n,\n",
     "        position_wise=False,\n",
     "        n_anchors=n_anchors,\n",
-    "        n_times=params[\"n_times\"],\n",
+    "        n_times=np.Infinity if one_per_time else params[\"n_times\"],\n",
```

Comment:
Is there a reason you use np.Infinity and not something more common like "None"?

Reply:
Ok makes sense!